app.rb

```diff
@@ -0,0 +1,32 @@
+require 'sinatra/base'
+require './lib/user'
+
+
+class ChitterApp < Sinatra::Base
+  enable :sessions
+
+  get '/' do
+    erb :homepage
+  end
+
+  get '/signup' do
+    erb :signup
+  end
+
+  post '/signup' do
+      User.create(
+      name: params[:name],
+      username: params[:username],
+      email: params[:email],
+      psw: params[:psw]
+    )
+    redirect '/chitter'
+  end
+
+  get '/chitter' do
+    @current_user
```

Comment:
this instance variable doesn't get passed to this view - maybe refactor to store it in session?